Source/WebCore/page/csp/ContentSecurityPolicy.cpp

```diff
@@ -328,7 +328,7 @@ bool ContentSecurityPolicy::urlMatchesSelf(const URL& url, bool forFrameSrc) con
 
 bool ContentSecurityPolicy::allowContentSecurityPolicySourceStarToMatchAnyProtocol() const
 {
-    if (is<Document>(m_scriptExecutionContext))
+    if (is<Document>(m_scriptExecutionContext.get()))
```

Comment:
Should we change this to `dynamicDowncast`, so we don't do the `is<Document>` check twice (once here, once in the downcast for its ASSERT)?

Reply:
Sure.